template/template.go

```diff
@@ -166,7 +166,7 @@ func NewTemplateExpander(
 				return html_template.HTML(text)
 			},
 			"match":     regexp.MatchString,
-			"title":     strings.Title, //nolint:staticcheck // TODO(beorn7): Need to come up with a replacement using the cases package.
+			"title":     strings.Title,
```

Comment:
Why is this different from upstream, was there a mistake when resolving a conflict?

Upstream is like this:
```suggestion
			"title":     cases.Title(language.AmericanEnglish, cases.NoLower).String,
```

Reply:
This was fixed in a later PR but it failed linting here. I didn't go up to the HEAD yet because there are too many conflicts to resolve and it was a bit of a mess

Reply:
I will push ahead and get this in the next bump